client/galaxy/scripts/mvc/tag.js

```diff
@@ -133,7 +135,70 @@ var TagsEditor = Backbone.View
 });
 
 // =============================================================================
+
+var WorkflowTagsEditor = TagsEditor.extend({
```

Comment:
Any chance we can re-use or extend the tags editor without introducing a workflow specific tag editor?

Reply:
@guerler I merged the WorkflowTagsEditor with the TagsEditor in https://github.com/galaxyproject/galaxy/pull/4476/commits/6d4441955ad7866308729cff37267643f8fa9fb9, is that what you had in mind ?